src/main/java/pages/OdometerReadingPage.java

```diff
@@ -0,0 +1,134 @@
+package pages;
+
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class OdometerReadingPage extends BasePage {
+
+    private static final String ODOMETER_READING_PAGE_TITLE = "Odometer reading";
+    private static final String READING_TITLE = "READING";
+    private static final String UNIT_TITLE = "UNIT";
+    private static final String READING_FIELD_CLASS_NAME = "XCUIElementTypeTextField";
+    private static final String BUTTONS_CLASS_NAME = "XCUIElementTypeButton";
+
+    private static final String KILOMETRES_TEXT = "Kilometres";
+    private static final String MILES_TEXT = "Miles";
+    private static final String SAVE_BUTTON_ID = "Save";
+    private static final String KILOMETRES_OPTION_ID = "Kilometres";
+    private static final String MILES_OPTION_ID = "Miles";
+    private static final String CANCEL_OPTION_ID = "Cancel";
+
+
+    public boolean isUnitDisplayed(UnitOptions unitOptions) {
+        boolean status = false;
+        switch (unitOptions) {
+            case KILOMETRES:
+                status = findElementById(KILOMETRES_TEXT + " Edit").isDisplayed();
+                break;
+            case MILES:
+                status = findElementById(MILES_TEXT + " Edit").isDisplayed();
+                break;
+        }
+        return status;
+    }
+
+    public enum Units { KILOMETRES, MILES }
```

Comment:
please add for enum values, remove final static text for those and use getValue() in switch option

Reply:
done